tests/views/test_admin_committee_questions.py

```diff
@@ -124,6 +124,57 @@ def test_upload_committee_question_document_with_new_format(self):
         # Delete the question that was created
         self.created_objects.append(question)
 
+    def test_upload_committee_question_document_with_navigable_string_error(self):
+        """
+        Upload committee question document (/admin/committee-question/upload)
+        """
+        url = "/admin/committee-question/upload"
+        data = {}
+        path = self.get_absolute_file_path("../data/RNW1153-200619.docx")
```

Comment:
can we put it in a folder with the suite or test case name?

Reply:
Will do